src/dom-utils/getParentNode.js

```diff
@@ -1,16 +1,19 @@
 // @flow
 import getNodeName from './getNodeName';
+import getDocumentElement from './getDocumentElement';
 
-export default function getParentNode(element: Node | ShadowRoot): Node {
+export default function getParentNode(
+  element: Node | ShadowRoot
+): Node | Element {
```

Comment:
```suggestion
): Node {
```

`Element` is an interface of `Node`, we don't need to explicitly list it.